tools/find_stray_release_notes.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Utility script to verify qiskit copyright file headers"""
+
+import argparse
+import multiprocessing
+import subprocess
+import os
+import sys
+import re
+
+# release notes regex
+reno = re.compile(r"releasenotes\/notes")
+# exact release note regex
+exact_reno = re.compile(r"^releasenotes\/notes")
+
+def discover_files():
+    """Find all .py, .pyx, .pxd files in a list of trees"""
+    cmd = ["git", "ls-tree", "-r", "--name-only", "HEAD"]
+    res = subprocess.run(cmd, capture_output=True, check=True, encoding="UTF8")
+    files = res.stdout.split("\n")
+    return files
+
+
+def validate_path(file_path):
+    if reno.search(file_path) and not exact_reno.search(file_path):
+        return file_path
+    return None
+
+
+def _main():
+    default_path = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "qiskit"
+    )
+    parser = argparse.ArgumentParser(description="Find any stray release notes.")
+    args = parser.parse_args()
+    files = discover_files()
+    with multiprocessing.Pool() as pool:
```

Comment:
This works in every platform?

Reply:
Yeah it will, because I have the `if __name__` = "__main__"` on the script. Without this windows and macOS > 3.7 would potentially have issues because they default to the spawn method.